video/opengl.h

```diff
@@ -21,14 +21,12 @@ class OpenGL : public Input
     /*!
      * \brief Create a Video::Input for reading from an OpenGL window
      *
-     * @param readX, readY The starting coordinates to read from (from bottom left of screen)
-     * @param readWidth, readHeight Size of image
-     *
-     * **NOTE** intentionally NOT using a cv::Rect as OpenGL coordinates are (typically)
-     * from bottom left of screen which would require window size etc. to convert
+     * @param size Size of image
+     * @param bottomLeft The starting coordinates to read from (from bottom left of screen)
      */
-    OpenGL(GLint readX,  GLint readY,  GLsizei readWidth,  GLsizei readHeight)
-      : m_ReadX(readX), m_ReadY(readY), m_ReadWidth(readWidth), m_ReadHeight(readHeight)
+    OpenGL(const cv::Size &size, const cv::Point &bottomLeft = { 0, 0 })
```

Comment:
I think this is a good change but can you check that ArdinMB and the various examples are updated to handle it?

Reply:
Ok, will do!